manifests/deploy/openshift/olm/metering.operatorgroup.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: operators.coreos.com/v1
 kind: OperatorGroup
 metadata:
   name: metering-operators
-  namespace: openshift-metering
+  namespace:
```

Comment:
@timflannagan1 hmmm...don't know why this is empty ...will need to take a look again what I messed up